src/Tickets/Admin/Onboarding/views/checklist-section/step-list/list-item.php

```diff
@@ -0,0 +1,35 @@
+<?php
+/**
+ * Step list item template.
+ *
+ * @since TBD
+ *
+ * @var \Tribe\Tickets\Admin\Onboarding\Template  $this      The template instance.
+ * @var \Tribe\Tickets\Admin\Onboarding\Installer $installer The installer instance.
+ *
+ * @var string  $id            The ID of the step list item.
+ * @var array   $classes       The classes for the step list item.
+ * @var string  $title         The title of the step list item.
+ * @var string  $link          The link for the step list item.
+ * @var string  $link_text     The text for the step list item link.
+ */
+
+$classes = array_merge( $classes, [ 'step-list__item' ] );
+if ( $id === 'tec-tickets-onboarding-wizard-events-item' ) {
+	$foo = 'bar';
+}
```

Comment:
??

Reply:
Whups - was using it fr a breakpoint, thought I got them all.